src/Microsoft.Extensions.ObjectPool/DefaultObjectPool.cs

```diff
@@ -56,19 +56,18 @@ public override T Get()
         private T GetViaScan()
         {
             ObjectWrapper[] items = _items;
-            T item = null;
 
             for (var i = 0; i < items.Length; i++)
             {
-                item = items[i];
+                T item = items[i];
```

Comment:
nit: var

Reply:
Since this is for the patch, I'm sticking to correctness changes 😆

Reply:
I agree, it is more correct to not use `var`.

trollololol

Reply:
You know what I mean 😑 - we have a different standard for patches